### ISSUE TITLE
Please provide support for the Lambda container image

### ISSUE THREAD

**Issue by adilnaimi** [aws-samples/parameters-secrets-lambda-extension-sample#2]
I'm looking for information on using the extension in a container image for Lambda. Does it support Lambda deployed as a container image?

**Comment by pulp-0011** [comment]
Certainly, the extension can be utilized within a Lambda container image. However, as Lambda functions packaged as container images do not support adding Lambda layers to the function configuration, there are additional steps involved. I recommend following this blog [Working with Lambda layers and extensions in container images](https://aws.amazon.com/blogs/compute/working-with-lambda-layers-and-extensions-in-container-images/) to understand various approaches to use the functionality of Lambda layers and extension with container images. 

You can obtain the extension file to add to your Dockerfile using below commands.

Replace the region and ARN value with your region and the[ matching ARN](https://docs.aws.amazon.com/systems-manager/latest/userguide/ps-integration-lambda-extensions.html#ps-integration-lambda-extensions-add) to retrieve extension file

```
aws lambda get-layer-version-by-arn \
  --region us-east-1 \
  --arn arn:aws:lambda::layer:AWS-Parameters-and-Secrets-Lambda-Extension:00
```

The value returned for _**Location**_ in the response, is the pre-signed URL of the S3 bucket that contains the extension. Download the .zip file from URL using curl/wget or from your web browser.

`curl -o parameter-secrets-extension.zip "S3-Bucket-Location-URL" `
